.github/workflows/release.yaml

```diff
@@ -142,11 +145,13 @@ jobs:
           title: 'Gingersnap Operator ${{ env.release-version }}'
           body: 'Gingersnap Operator ${{ env.release-version }}'
 
-      - name: Trigger Publish to main job
+      - name: Trigger Post Release job
         uses: cardinalby/schedule-job-action@v1
         with:
          ghToken: ${{ secrets.PAT_TOKEN }}
          targetBranch: 'main'
          templateYmlFile: './operator/.github-scheduled-workflows/publish-main.yaml'
          targetYmlFileName: publish-main.yaml
-
+        env:
```

Comment:
I think we need to set `copyEnvVariables: true` in order for the environment variables to be propagated to the scheduled job.